airflow/serialization/serialized_objects.py

```diff
@@ -53,14 +55,23 @@
 log = logging.getLogger(__name__)
 FAILED = 'serialization_failed'
 
-BUILTIN_OPERATOR_EXTRA_LINKS: List[str] = [
-    "airflow.providers.google.cloud.operators.bigquery.BigQueryConsoleLink",
-    "airflow.providers.google.cloud.operators.bigquery.BigQueryConsoleIndexableLink",
-    "airflow.providers.google.cloud.operators.mlengine.AIPlatformConsoleLink",
-    "airflow.providers.qubole.operators.qubole.QDSLink",
+OPERATOR_EXTRA_LINKS: Set[str] = {
```

Comment:
```suggestion
_OPERATOR_EXTRA_LINKS: Set[str] = {
```
I don't want anyone to accidentally use this variable instead of calling functions `get_operator_extra_links`.

Reply:
Good idea.